src/graphql/schema.gql

```diff
@@ -3,7 +3,7 @@ scalar Any
 # Queries
 
 type Query {
-  space(id: String): Space
+  space(id: String!): Space
```

Comment:
I think with this we can remove this line https://github.com/snapshot-labs/snapshot-hub/blob/master /src/graphql/operations/space.ts#L6 and for vote https://github.com/snapshot-labs/snapshot-hub/blob/master /src/graphql/operations/vote.ts#L7